vikitaotiz/iReporter
# [#162116362] Create Sign Up Page

**What does this PR do?**
It Creates a sign up page for a user.

**Description of Task to be completed?**
The user should be able to login using a valid first and last name, Email Address and Password. If the credentials are false there is an error to the system.

**How should this be tested?**
Clone the repository to you local machine then access the page using your local URL to your a browser. Try to log in.

Background context you want to provide?
It is a requirement with the project guidelines that the credentials should be validated.

**What are the relevant pivotal tracker stories?**
#162116362

Relevant Screenshots
![signup](https://user-images.githubusercontent.com/42869046/48844595-6422e880-edab-11e8-9ff9-6227c9e9c8e1.PNG)
